.eslintrc.cjs

```diff
@@ -191,6 +191,13 @@ const config = {
         'no-console': 'off',
       },
     },
+    {
+      files: './dbschema/seeds/*.ts',
+      rules: {
+        'import/no-default-export': 'off',
+        // 'import/no-named-export': 'error',
```

Comment:
Is the comment left intentionally?

Reply:
Yeah I thought about restricting seeds to not export anything but their own default execution function.
But then I thought that some seed helpers might emerge from sister files, so I didn't want to primitively block that.